src/eval_framework/llm/vllm.py

```diff
@@ -428,6 +428,51 @@ def seq_length(self) -> int | None:
         return self.max_seq_length
 
 
+class _VLLM_from_wandb_registry(VLLMModel):
+    """
+    A class to create VLLM instances from registered models in Wandb registry.
+    Downloads the model artifacts from Wandb and creates a local VLLM instance.
+    """
+
+    LLM_NAME = ""
+
+    def __init__(
+        self,
+        artifact_name: str,
+        version: str = "latest",
+        formatter: str = "",
```

Comment:
Isn't it confusing that in the base class `VLLMModel`, it's a different type: `formatter: BaseFormatter | None = None`, thus works differently?

Reply:
Nah, makes sense. Resolving.